docs/scheduling_autoscaling.md

```diff
@@ -0,0 +1,53 @@
+# Scheduling and Autoscaling
+
+⚠️⚠️⚠️ **This is currently a development feature and has not been released** ⚠️⚠️⚠️
+
+> Autoscaling is currently ongoing work within Agones. The work you see here is just the beginning.
+
+Scheduling and autoscaling tend to go hand in hand, as where in the cluster `GameServers` are provisioned
+tends to impact how to autoscale fleets up and down (or if you would even want to)
+
+## Fleet Autoscaling
+
+Fleet autoscaling is currently the only type of autoscaling that exists in Agones. It is also only available as a simple
+buffer autoscaling strategy. Have a look at the [Create a Fleet Autoscaler](create_fleetautoscaler.md) quickstart,
+and the [Fleet Autoscaler Specification](fleetautoscaler_spec.md) for details.
+
+Node scaling, and more sophisticated fleet autoscaling will be coming in future releases ([design](https://github.com/GoogleCloudPlatform/agones/issues/368))
+
+## Fleet Allocation Scheduling
+
+There are two scheduling strategies for fleets - each designed for different types of Kubernetes Environments.
+
+### Packed
+
+This is the *default* Fleet scheduling strategy. It is designed for dynamic Kubernetes environments, wherein you wish 
+to scale up and down as load increases or decreases, such as in a Cloud environment where you are paying
+for the infrastructure you use.
+
+It attempts to _pack_ as much as possible into the smallest set of nodes, to make
+scaling infrastructure down as easy as possible.
+
+Currently, Allocation scheduling is the only aspect this strategy affects, but in future releases it will
+also affect `GameServer` `Pod` scheduling, and `Fleet` scale down scheduling as well.
```

Comment:
`allocation scheduling` is difficult to grasp, I do understand but I'm not sure everyone will. Should we explain or refer to what allocation is ? Or give concrete example.

Reply:
That's a fair comment - I'll take another pass, see if I can add some more to make it clearer.

Reply:
Updated - hopefully this is clearer.